core/src/main/dml/messaging.dml

```diff
@@ -18,6 +18,7 @@ class MessagingSystem {
 }
 
 class Sender {
+    Boolean htmlSender;
```

Comment:
Given that you are treating a `null` value here as `false`, you should just use a primitive `boolean`.